examples/src/scala/org/pantsbuild/example/several_scala_targets/greeting_by_name.scala

```diff
@@ -0,0 +1,8 @@
+// Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+object GreetingByName {
+  def greeting_by_name(name: String): String = {
+    "Hello %s! How are you?".format(name)
```

Comment:
Super nit: 

Scala has really cool string interpolation: `s"Hello {name}! How are you?"` (notice the `s"` at the beginning of the string).